utils/cups-browsed.c

```diff
@@ -694,6 +697,27 @@ stop_debug_logging()
   lfp = NULL;
 }
 
+// returns the size of debug log file
+long int findLogFileSize() 
+{ 
+    FILE* fp = fopen(debug_log_file, "r"); 
+    if (fp == NULL) { 
+        return -1; 
+    } 
+    fseek(fp, 0L, SEEK_END); 
+    long int res = ftell(fp); 
+    fclose(fp); 
+    return res; 
+}
+
+void copyToFile(FILE **fp1, FILE **fp2){
+  char ch;
```

Comment:
Is the file copying not faster if you read and write bigger chunks, like 2048 bytes?

Reply:
Yes it is I will implement copying between files using chunk method then.